src/routers/CancelMatchRouter.ts

```diff
@@ -0,0 +1,24 @@
+import { Request } from 'express';
+import Constants from '../common/constants';
+import MatchRepo from '../repos/MatchRepo';
+import AuthenticatedApplicationRouter from '../utils/AuthenticatedApplicationRouter';
+
+class UpdateMatchAvailabilitiesRouter extends AuthenticatedApplicationRouter<void> {
+  constructor() {
+    super('POST');
+  }
+
+  getPath(): string {
+    return '/cancel/';
+  }
+
+  async content(req: Request): Promise<void> {
+    const { matchID } = req.query;
+    if (!matchID) throw Error('Missing required matchID');
+
+    const match = await MatchRepo.getMatchByID(matchID);
+    await MatchRepo.updateMatch(match, { status: Constants.MATCH_CANCELED });
```

Comment:
would it be worth considering the case where someone tries to cancel the match after the cron has set the most recent match to inactive but not yet assigned a new match (therefore causing a match that is inactive to be converted to canceled and losing that match in the user's history)

Reply:
hmm, I presume it's not necessary only because this would require an explicit call to the `match/cancel` endpoint, which needs a matchID. However, I will add in this logic to make sure every case is handled.